src/components/containers/CreateAccount/CreateAccount.js

```diff
@@ -88,6 +91,15 @@ const CreateAccount = ({
     return points.length >= AppInfo.minEntropyLength
   }
 
+  const walletValidity = (wallets) => {
+    setAccountWalletValid(wallets.length > 0)
```

Comment:
If `wallets.length` is smaller than 1, JS already evaluates it as "falsy". And if it is greater than 0 JS will evaluate it as "truthy".
So the condition `wallets.length > 0`  should be not necessary.